README.md

```diff
@@ -20,20 +20,62 @@ Language mappings are defined in `.tx/config`, with the second code being the on
 
 # Building
 
-### Dependencies
-- Android SDK, with `$ANDROID_HOME` pointing to it (you can skip this if you are using Android Studio)
-- Android NDK (you should install the required version (`ext.ndkVersionShared` in `./build.gradle`) with the usual tools, such that it's located within `$ANDROID_HOME/ndk/`).
-- Go (see [here](https://docs.syncthing.net/dev/building.html#prerequisites) for the required version)
-- Java Version 11 (you scan skip this if you are using Android Studio, otherwise you might need to set `$JAVA_HOME` accordingly)
+## Dependencies
 
-### Build instructions
+These are necessary for building from the command line. If you build using
+Docker, you don't need to set them up separately.
```

Comment:
Should probably be moved directly below `# Building`, and also mention using android studio as an option. In both cases all the sections aren't relevant.

Reply:
Done.